Tests/TestUtils/ReupSceneInstantiator.cs

```diff
@@ -109,6 +111,8 @@ private static SceneObjects SceneObjectsWithBuilding(GameObject building)
 
             MoveDhvCamera moveDhvCamera = dollhouseViewWrapper.GetComponent<MoveDhvCamera>();
 
+            ZoomDhvCamera zoomDhvCamera = dollhouseViewWrapper.GetComponent<ZoomDhvCamera>();
```

Comment:
```suggestion
            MoveDhvCamera moveDhvCameraBehavior = dollhouseViewWrapper.GetComponent<MoveDhvCamera>();

            ZoomDhvCamera zoomDhvCameraBehavior = dollhouseViewWrapper.GetComponent<ZoomDhvCamera>();
```
Again, let's try to make this variables name as expressive as we can so we can avoid future confusion. 
The same for the actual properties in the `SceneObjects` which are the actual names that the tests see